qt/RpcClient.h

```diff
@@ -33,6 +34,16 @@ extern "C"
   struct tr_session;
 }
 
+struct RpcResponse
```

Comment:
May be one of those places where in-place initialization is a nice feature to use (instead of declaring the constructor).